modules/guardduty-member-account/README.md

```diff
@@ -0,0 +1,48 @@
+# terraform-aws-tardigrade-guardduty/org_admin_account
```

Comment:
I think this directory can be renamed just `member`. We're already in the guardduty module, so naming it `guardduty-member-account` is a bit redundant.

```suggestion
# terraform-aws-tardigrade-guardduty/member
```

Reply:
Renamed the submodule directory names.  Also updated the source path in the associated test modules.